baseplate/integration/mixins/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .lazy_attributes import LazyAttributesMixin
```

Comment:
how about just putting this in `baseplate/integration/__init_.py` and prefixing the class name with `_` to mark it private?

i don't think we really need a whole subpackage for this one thing.